index.html

```diff
@@ -73,7 +73,298 @@ <h1>I'm a software developer</h1>
+        </div>
+      </div>
+      <!-- end of sixth card  -->
+      <!-- *end of work -->
+      <!-- start of about me -->
+
+      <div class="resume">
+        <div class="h2">
+          <h2>About me</h2>
+        </div>
+        <div class="p">
+          <p>
+            I can help you build a product,<br />
+            feature or website. Look through<br />
+            some of my work and experience! If<br />
+            you like what you see and have a<br />
+            project you need coded,<br />
+            don't hesitate to contact me.<br />
+          </p>
+        </div>
+        <button class="res-btn">Get My Resume</button>
+      </div>
+      <div class="languages">
+        <div class="lang-btn">
+          <h2>Languages</h2>
+          <a href="#"
+            ><img
+              class="javascr-btn"
+              src="images/languages/JavaScript.png"
+              alt=""
+          /></a>
+          <a href="#"
+            ><img
+              class="ruby-btn"
+              src="images/languages/Ruby.png"
+              alt="ror-button"
+          /></a>
+          <a href="#"
+            ><img
+              class="html-btn"
+              src="images/languages/Html.png"
+              alt="ror-button"
+          /></a>
+          <a href="#"
+            ><img
+              class="css-btn"
+              src="images/languages/CSS.png"
+              alt="ror-button"
+          /></a>
+        </div>
+      </div>
+      <div class="frameworks">
+        <div class="frame-btn">
+          <h2>Frameworks</h2>
+          <a href="#"
+            ><img class="bt-btn" src="images/frameworks/Bootstrap.png" alt=""
+          /></a>
+          <a href="#"
+            ><img
+              class="rubyor-btn"
+              src="images/frameworks/Ruby on rails.png"
+              alt=""
+          /></a>
+          <a href="#"
+            ><img class="rspec-btn" src="images/frameworks/RSpec.png" alt=""
+          /></a>
+          <a href="#"
+            ><img class="capy-btn" src="images/frameworks/Capybara.png" alt=""
+          /></a>
+          <a href="#"
+            ><img class="sel-btn" src="images/frameworks/Selenium.png" alt=""
+          /></a>
+        </div>
+      </div>
+      <div class="skills">
+        <div class="skills-btn">
+          <h2>Skills</h2>
+          <a href="#"
+            ><img class="ckit-btn" src="images/skills/Codekit.png" alt=""
+          /></a>
+          <a href="#"
+            ><img class="gith-btn" src="images/skills/Github.png" alt=""
+          /></a>
+          <a href="#"
+            ><img class="cpen-btn" src="images/skills/Codepen.png" alt=""
+          /></a>
+          <a href="#"
+            ><img class="gitl-btn" src="images/skills/Gitlab.png" alt=""
+          /></a>
+          <a href="#"
+            ><img class="term-btn" src="images/skills/Terminal.png" alt=""
+          /></a>
+        </div>
+      </div>
```

Comment:
- [ ] Your design here is awesome :100:, but it is required to make technology in each category(e.g., "JavasScript", "HTML" and "CSS") a `ul` with each being inside `li`, And also you need to follow the same structure for technology categories ("Languages", "Frameworks" and "Skills"). Please make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li` :+1: